assets/templates/cards-component.html

```diff
@@ -150,14 +176,21 @@ <h3>Plan name</h3>
           </div>
           <div>
             <h3 class="dpsg-tit-card m-l-30">PLAN DESTACADO</h3>
-            <div class="dp-card-ribbon-rt" data-ribbon="Recomendado">
-              <h3>Plus</h3>
-              <p>¿Realizas envíos periodicamente o un newsletter al mes?</p>
-              <p>Hasta 100.000 Contactos.</p>
-              <div class="dp-price" data-start="Desde">
-                <div class="dp-amount">
-                  <span class="dp-money-number" data-money="US$">5250</span>
-                  <span>por mes</span>
+            <div class="dp-card-ribbon-rt">
+              <div class="dp-ribbon">
+                <span class="dp-text-ribbon">Recomendado</span>
+              </div>
+              <div class="dp-content-plans">
```

Comment:
Is it really neccesary to add an extra wrapper div for all extra except the ribbon?

Reply:
I'm working on separating the highlighted state from the ribbon